### PR TITLE
Added Empty Definitions For Missing Getters And Mutations

### LINKED ISSUE #2195: 🐛 [Bug] console.errors in /webapp
> [mattwr18](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-11-11T18:59:19Z" title="Monday, November 11th 2019, 7:59:19 pm +01:00">Nov 11, 2019</time>_
_Closed <time datetime="2019-11-12T14:54:46Z" title="Tuesday, November 12th 2019, 3:54:46 pm +01:00">Nov 12, 2019</time>_
---

## :bug: Bugreport

Currently, our test suite is throwing these console.errors
in /pages/index.spec.js
```
 [vuex] unknown getter: posts/currentPosts
[vuex] unknown mutation type: posts/SET_CURRENT_POSTS
```
see https://github.com/Human-Connection/Human-Connection/blob/master/webapp/components/FilterPosts/FilterPosts.spec.js#L59 for an example of how to configures `mutations` and `getters` for a test suite

### Steps to reproduce the behavior
1. run `yarn test pages/index.spec.js --watch --notify` from `/webapp`
2. see console.errors
3. fix by configuring the tests

### PR DESCRIPTION
**Pull request by Tirokk**
> [Mogge](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2019-11-11T20:23:01Z" title="Monday, November 11th 2019, 9:23:01 pm +01:00">Nov 11, 2019</time>_
_Merged <time datetime="2019-11-12T14:54:46Z" title="Tuesday, November 12th 2019, 3:54:46 pm +01:00">Nov 12, 2019</time>_
---

### Issues
- fixes #2195

### CODE REVIEW

File: webapp/pages/index.spec.js [hunk 1]
```diff
@@ -53,6 +54,7 @@ describe('PostIndex', () => {
         'auth/user': () => {
           return { id: 'u23' }
         },
+        'posts/currentPosts': () => [],
```
Comment (Tirokk) [inline webapp/pages/index.spec.js:57]: > [mattwr18](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-11-11T23:39:32Z" title="Tuesday, November 12th 2019, 12:39:32 am +01:00">Nov 12, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -53,6 +54,7 @@ describe('PostIndex', () => {
         'auth/user': () => {
           return { id: 'u23' }
         },
+        'posts/currentPosts': () => jest.fn(),
```


```suggestion
        'posts/currentPosts': () => [],
```

Comment (Tirokk) [reply webapp/pages/index.spec.js:57]: > [Mogge](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2019-11-12T10:21:40Z" title="Tuesday, November 12th 2019, 11:21:40 am +01:00">Nov 12, 2019</time>_

---


done

**Review by Mogge — CHANGES_REQUESTED** [review]
Looks good @Mogge, I'd suggest returning an empty array, as that's he default

**Review by Mogge — CHANGES_REQUESTED** [review]
Looks good @Mogge, I'd suggest returning an empty array, as that's he default

**Review by Mogge — APPROVED** [review]
great job @Mogge !!
thanks a lot :blush:

**Pull request closed without merging**